[messenger] Adds a stamp to provide a routing key on message publishing (#30008)
symfony/symfony: UPGRADE-5.0.md

---------

 * The `LoggingMiddleware` class has been removed, pass a logger to `SendMessageMiddleware` instead.

>  * Messages aren't published anymore with a routing key from queue configuration. Use exchange configuration instead.

Reviewer: ```suggestion
 * Routing key from queue configuration has been removed. Use exchange configuration instead (`exchange[routing_key]` in DSN).
```
